cmd/xl-v1-metadata.go

```diff
@@ -315,6 +315,13 @@ func (m xlMetaV1) ToObjectInfo(bucket, object string) ObjectInfo {
 	// All the parts per object.
 	objInfo.Parts = m.Parts
 
+	// Update storage class
+	if sc, ok := m.Meta[amzStorageClass]; ok {
+		objInfo.StorageClass = sc
+	} else {
+		objInfo.StorageClass = globalMinioDefaultStorageClass
+	}
```

Comment:
@nitisht I guess it is okay for StorageClass to be empty in FS for the moment. Approving this.